Frameworks/UIKit/StarboardXaml/ApplicationCompositor.cpp

```diff
@@ -53,12 +53,31 @@ void InitializeApp() {
     SetupMainRunLoopTimedMultipleWaiter();
 }
 
+extern "C" void RunApplicationMainWithString(Platform::String^ principalClassName,
+                                   Platform::String^ delegateClassName,
+                                   float windowWidth,
+                                   float windowHeight,
+                                   ActivationType activationType,
+                                   Platform::String^ activationString) {
+    // Perform initialization
+    InitializeApp();
+
+    // Kick off iOS application main startup
+    ApplicationMainStartHSTRING(
```

Comment:
Great place for a comment explaining this.

---
In reply to: [68809747](https://github.com/Microsoft/WinObjC/pull/592#discussion_r68809747) [](ancestors = 68809747)